### PR TITLE
Add exception handling for exception logging

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

File: src/Hyperion/ValueSerializers/ObjectSerializer.cs [hunk 1]
```diff
@@ -103,7 +103,18 @@ public override void WriteManifest(Stream stream, SerializerSession session)
...
-        public override object ReadValue(Stream stream, DeserializerSession session) => _reader(stream, session);
+        public override object ReadValue(Stream stream, DeserializerSession session)
+        {
+            try
+            {
+                return _reader(stream, session);
+            }
+            catch (Exception e)
+            {
+                throw new Exception(
```
Comment (Aaronontheweb) [inline src/Hyperion/ValueSerializers/ObjectSerializer.cs:114]: Should this be a `SerializationException` or something more specific? Do we have any Hyperion-specific exception types?

**Review by Aaronontheweb — COMMENTED** [review]
Left a nit-picking remark

File: src/Hyperion/ValueSerializers/ObjectSerializer.cs (hunk 1 above)
Comment (Arkatufus) [reply src/Hyperion/ValueSerializers/ObjectSerializer.cs:114]: `SerializationException` doesn't exist in netstandard1.6 and net45

Comment (Arkatufus) [reply src/Hyperion/ValueSerializers/ObjectSerializer.cs:114]: I guess we can add `System.Runtime.Serialization.Primitives` package to add it in?

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Comment by Aaronontheweb** [comment]
Let's push out a new release with these changes included so we can try to track down https://github.com/akkadotnet/akka.net/issues/5102

**Pull request merged**